test/examples/ucp_client_server.c

```diff
@@ -411,12 +425,16 @@ static int init_worker(ucp_context_h ucp_context, ucp_worker_h *ucp_worker)
 static void server_conn_handle_cb(ucp_conn_request_h conn_request, void *arg)
 {
     ucx_server_ctx_t *context = arg;
-    ucp_ep_h         ep;
-    ucp_ep_params_t  ep_params;
-    ucs_status_t     status;
-    ucp_worker_h     data_worker;
 
-    data_worker = context->data_worker;
+    context->conn_request = conn_request;
```

Comment:
as we discussed, pls check context->conn_request == NULL

Reply:
how it supposed to be handled? if reject then it should be handled on client